NaviGator/mission_control/navigator_alarm/navigator_alarm_handlers/kill.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
```

Comment:
Shebangs will need to be updated to use Python 3 in case the user has `python` symlinked to Python 2.
```suggestion
#!/usr/bin/env python3
```